server/views/pages/exhibition.njk

```diff
@@ -35,89 +35,97 @@
-                <div class="flex flex--v-center font-elf-green {{ {s:'HNM4', m:'HNM5', l:'HNM4'} | fontClasses }}">
-                  {% icon 'other/ticket', null, ['icon--2x', 'icon--teal'] %}
-                  <strong class="caps {{ {s: 2} | spacingClasses({margin: ['left']}) }} {{ {s: 6} | spacingClasses({margin: ['right']}) }}">Free</strong>
-                  {% if exhibition.end %}
+              <div class="bg-white inline-block exhibition-hero__box {{ {m:3, l:4} | spacingClasses({'padding': ['right', 'left']}) }} {{ {s:4} | spacingClasses({'padding': ['top', 'bottom']}) }} rounded-top-left rounded-bottom-right">
+                <div class="flex flex--v-center flex--h-space-between {{ {s:'HNM4', m:'HNM5', l:'HNM4'} | fontClasses }}">
+                  <span class="flex flex--v-center">
+                    {% icon 'other/ticket', null, ['icon--2x'] %}
+                    <strong class="{{ {s:'WB7', l:'WB6'} | fontClasses }} caps {{ {s: 2} | spacingClasses({margin: ['left']}) }} {{ {s: 6} | spacingClasses({margin: ['right']}) }}">Free</strong>
+                  </span>
+                  <span class="flex flex--v-center">
+                    <span class="{{ {s:2} | spacingClasses({margin: ['right']}) }} flex flex--v-center">
+                      {% set statusColor = 'icon--' + {start: exhibition.start, end: exhibition.end} | formatDateRangeWithColor %}
+                      {% icon 'other/status-indicator', null, [statusColor] %}
+                    </span>
                     <strong>{{ {start: exhibition.start, end: exhibition.end} | formatDateRangeWithMessage }}</strong>
-                  {% endif %}
+                  </span>
                 </div>
-                <h1 class="{{ {s:'WB4', m:'WB5', l:'WB4', xl:'WB3'} | fontClasses }}">{{ exhibition.title }}</h1>
-                <span class="{{ {s:'HNM4', m:'HNM5', l:'HNM3'} | fontClasses }}">
+                <h1 class="{{ {s:'WB5', m:'WB6', l:'WB4', xl:'WB3'} | fontClasses }}">{{ exhibition.title }}</h1>
+                <span class="{{ {s:'HNL4', m:'HNL5', l:'HNL3', xl:'HNL2'} | fontClasses }}">
                   {% if exhibition.end %}
                     <time datetime="{{ exhibition.start }}">{{ exhibition.start | formatDate }}</time> &ndash; <time datetime="{{ exhibition.end }}">{{ exhibition.end | formatDate }}</time>
                   {% else %}
                     Permanent exhibition
                   {% endif %}
                 </span>
               </div>
-
             </div>
           </div>
         </div>
-
       </div>
     </div>
 
 
-    {# {% componentV2 'image', exhibition.featuredImage, {}, {lazyload: true} %} #}
     <div class="is-hidden-s">
-      {% component 'wobbly-edge', {background: 'white'} %}
+      {% component 'wobbly-edge', {background: 'white', intensity: 30} %}
```

Comment:
What's intensity?

Reply:
It controls the maximum height of the wobbly line. Or how wobbly the wobbly line is, if you like.